go.mod

```diff
@@ -4,7 +4,7 @@ go 1.17
 
 require (
 	github.com/google/uuid v1.3.0
-	github.com/scrapli/scrapligo v1.0.0
+	github.com/scrapli/scrapligo v1.0.2-0.20220703215242-e7bff0960e85
```

Comment:
I will need to bemp it to 1.0.3(4)

Reply:
tagging 1.1.0 here in a second for ya!

Reply:
done in [e8f2581](https://github.com/carlmontanari/boxen/pull/33/commits/e8f25814ac69b8ca80599e7ef4be4c550472b564)